docs/docs/graphql-api.md

```diff
@@ -0,0 +1,266 @@
+  ) => (
+    <header>
+      <h1>{data.site.siteMetadata.title}</h1>
+    </header>
+  )}
+/>
+```
+
+#### Returns
+
+The StaticQuery component returns `data` in a `render` prop:
+
+```jsx
+<StaticQuery
+  // ...
+  // highlight-start
+  render={data => (
+    <header>
+      <h1>{data.site.siteMetadata.title}</h1>
+    </header>
+  )}
+  // highlight-end
+/>
+```
+
+### `useStaticQuery`
+
+The `useStaticQuery` hook can be used similar to `StaticQuery` in any component or page, but doesn't require the use of a component and render prop.
+
+Because it is a React hook, the [rules of hooks](https://reactjs.org/docs/hooks-rules.html) apply and you'll need to use it with React and ReactDOM version 16.8.0 or later. Because of how queries currently work in Gatsby, only one instance of `useStaticQuery` is supported in each file.
+
+Also, refer to the [guide on querying data in components with useStaticQuery](/docs/use-static-query/).
+
+#### Params
+
+The `useStaticQuery` hook takes one argument:
+
+- `query`: a `graphql` query string
+
+```jsx
+const data = useStaticQuery(graphql`
+  query HeaderQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
+`)
+```
+
+#### Returns
+
+The `useStaticQuery` hook returns data in an object:
+
+```jsx
+const data = useStaticQuery(graphql`
+  query HeaderQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
+`)
+return (
+  // highlight-start
+  <header>
+    <h1>{data.site.siteMetadata.title}</h1>
+  </header>
+  // highlight-end
+)
+```
+
+## Query structure
+
+Queries are written in the same shape you want data returned in. How you source data will determine the names of fields that you can query on, based on the nodes they add to the GraphQL schema.
+
+For understanding the parts of a query refer to the [conceptual guide](/docs/graphql-concepts/#understanding-the-parts-of-a-query).
+
+### GraphQL query arguments
+
+GraphQL queries can take arguments to alter how the data is returned. The logic for these arguments is handled internally by Gatsby. Arguments can be passed in to fields at any level of the query.
+
+Different nodes can take different arguments based off of the nature of the node.
+
+The arguments you can pass to collections (like arrays or long lists of data - ex. `allFile`, or `allMdx`) are:
+
+- [`filter`](/docs/graphql-reference#filter)
+- [`limit`](/docs/graphql-reference#limit)
+- [`sort`](/docs/graphql-reference#sort)
+- [`skip`](/docs/graphql-reference#skip)
+
+The arguments you can pass to a `date` field are:
+
+- [`formatString`](/docs/graphql-reference#dates)
+- [`locale`](/docs/graphql-reference#dates)
+
+The arguments you can pass to an `excerpt` field are:
+
+- [`pruneLength`](/docs/graphql-reference#excerpt)
+- [`truncate`](/docs/graphql-reference#excerpt)
+
+### Graphql query operations
+
+Other built-in configurations can be used in queries
+
+- [`Alias`](/docs/graphql-reference#alias)
+- [`Group`](/docs/graphql-reference#group)
+
```

Comment:
```suggestion

For examples, refer to the [Query recipes](/docs/recipes/#6-querying-data) and [GraphQL Query options Reference Guide](/docs/graphql-reference/).
```
This URL will change slightly when we break the recipes up into pages, but we'll do a global find and replace to make sure there aren't broken links.